core/services/ocrcommon/block_translator.go

```diff
@@ -19,9 +19,9 @@ type BlockTranslator interface {
 // NewBlockTranslator returns the block translator for the given chain
 func NewBlockTranslator(cfg Config, client evmclient.Client, lggr logger.Logger) BlockTranslator {
 	switch cfg.ChainType() {
-	case chains.Arbitrum:
+	case config.Arbitrum:
 		return NewArbitrumBlockTranslator(client, lggr)
-	case chains.XDai, chains.ExChain, chains.Optimism:
+	case config.XDai, config.ExChain, config.Optimism:
```

Comment:
If this is too opaque now, we could change to something more idiomatic:
```suggestion
	case config.ChainTypeXDai, config.ChainTypeExChain, config.ChainTypeOptimism:
```
```suggestion	
	case config.ChainXDai, config.ChainExChain, config.ChainOptimism:
```